tests/test_session_2_byref_true.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+# Author: Leonardo Gama (@leogama)
+# Copyright (c) 2008-2016 California Institute of Technology.
+# Copyright (c) 2016-2022 The Uncertainty Quantification Foundation.
+# License: 3-clause BSD.  The full license text is available at:
+#  - https://github.com/uqfoundation/dill/blob/master/LICENSE
+
+from test_session_1_byref_false import *
+
+if __name__ == '__main__':
+    byref = True
+    dill.load_session(session_file % byref)
+    os.remove(session_file % byref)
```

Comment:
should guard the `os.remove` in case the file is missing or does not exist

Reply:
Ibidem.